projects/control-service/projects/pipelines_control_service/src/integration-test/java/com/vmware/taurus/datajobs/it/DataJobDeploymentCrudIT.java

```diff
@@ -77,4 +77,9 @@ protected void beforeDeploymentDeletion() throws Exception {
         .andExpect(status().isOk())
         .andExpect(jsonPath("$.vdk_version", is("release")));
   }
+
+  @Override
+  protected void afterDeploymentDeletion() {
```

Comment:
Presuably the rows wouldn't be in the database here either? 
Maybe we could just have the implementation in the base class.